src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -26,6 +23,12 @@ const defaultProps = {
 	selectedIndices: [],
 };
 
+const nonPassthrouhgs = [
```

Comment:
typo. should be "nonPassthroughs" but we may want to use "nonPassThroughs" for camel case consistency.